native/src/components/PoiPage.tsx

```diff
@@ -72,10 +72,7 @@ const PoiPage = ({
   const { location } = poi.location
   const files = resourceCache[poi.path] || {}
 
-  const navigationUrl =
-    location && poi.featureLocation?.geometry.coordinates
-      ? getNavigationDeepLinks(location, poi.featureLocation.geometry.coordinates)
-      : null
+  const navigationUrl = location && getNavigationDeepLinks(poi.location)
```

Comment:
No need to check for `location` here
```suggestion
  const navigationUrl = getNavigationDeepLinks(poi.location)
```

Reply:
I only found usages of LocationModel within the api-client